app/src/components/main/modal/modalLayout/modalFooter/modalFooter.jsx

```diff
@@ -110,16 +111,11 @@ export class ModalFooter extends Component {
               </BigButton>
             </div>
           )}
-          {multiActionOkButton && (
+          {customButton && (
             <div className={cx('button-container')}>
-              <MultiActionButton
-                onClick={onClickOk}
-                items={multiActionOkButton.items}
-                title={multiActionOkButton.title}
-              />
+              <customButton.component {...customButton.buttonProps} onClick={onClickOk} />
```

Comment:
I think instead of passing `onClickOk` we can pass `closeModal` function itself.

Reply:
Done